resources/home/dnanexus/utils/query.py

```
@@ -0,0 +1,487 @@
+                # If sample doesn't exist yet just add the data as is
+                cnv_data[sample]["sample"] = sample
+                cnv_data[sample]["Alignment BAM"] = data["Alignment BAM"]
+                cnv_data[sample]["Alignment BAI"] = data["Alignment BAI"]
+                cnv_data[sample]["CNV visualisation"] = data[
+                    "CNV visualisation"
+                ]
+                cnv_data[sample]["clinical_indications"][clin_ind][
+                    "CNV"
+                ].append(cnv_info)
+
+            except Exception as exc:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)